prac_05/emails.py

```diff
@@ -0,0 +1,31 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# Dictionary of Australian state codes to their full names
+CODE_TO_NAME = {
```

Comment:
The dictionary CODE_TO_NAME is well-defined and provides clear mappings from state codes to their full names.
Well you can also consider adding a comment above the dictionary to explain its purpose and providing more context for someone unfamiliar with the code.